src/components/Tokens/TokenDetails/StatsSection.tsx

```diff
@@ -34,11 +34,11 @@ const NoData = styled.div`
 
 type NumericStat = number | undefined | null
 
-function Stat({ value, title }: { value: NumericStat; title: ReactNode }) {
+function Stat({ value, title, isPrice = false }: { value: NumericStat; title: ReactNode; isPrice?: boolean }) {
   return (
     <StatWrapper>
       {title}
-      <StatPrice>{value ? formatDollarAmount(value) : '-'}</StatPrice>
+      {isPrice ? <StatPrice>{formatDollarPrice(value)}</StatPrice> : <StatPrice>{formatDollarAmount(value)}</StatPrice>}
```

Comment:
should formatDollar take a "price" arg as well?

Reply:
yeah was just thinking that. i don't think it is necessary to do this PR but you do duplicate the component in this ternary

Reply:
sg i did it here